### PR TITLE
added structured 11_deploy_keras_ai_platform_babyweight

### PR DESCRIPTION
**Pull request by ryangillard**

### CODE REVIEW

File: courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb [hunk 1]
```diff
@@ -0,0 +1,339 @@
+{
```
Comment (munnm) [inline courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>(personal preference) remove 1st paragraph, second paragraph suffices. Though fine how it is. Or maybe just remove "in this lab" from 2nd paragraph. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (munnm) [inline courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>remove export</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (munnm) [inline courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>if we're defining PROJECT above do we need to do it here too?</p><p> </p><p>Also, should there be TODOs or is the task just to replace Project and run cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (munnm) [inline courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>add PROJECT variable</p><p>os.environ["PROJECT"] = PROJECT</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (munnm) [inline courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>is this line too long. Not sure the conventions for bash.</p><p>Would "gsutil ls -d gs://${BUCKET}/babyweight/trained_model/2* | tail -1" suffice here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

File: courses/machine_learning/deepdive2/structured/solutions/5b_deploy_keras_ai_platform_babyweight.ipynb [hunk 2]
```diff
@@ -0,0 +1,390 @@
+{
```
Comment (munnm) [inline courses/machine_learning/deepdive2/structured/solutions/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>remove last paragraph</p><p> </p><p>Also (personal preference) first paragraph seems redundant. The second paragraph says it all well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (munnm) [inline courses/machine_learning/deepdive2/structured/solutions/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>remove export here as previously discussed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (munnm) [inline courses/machine_learning/deepdive2/structured/solutions/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>add os.environ["PROJECT"] = PROJECT</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (munnm) [inline courses/machine_learning/deepdive2/structured/solutions/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>line too long?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

**Comment by munnm** [comment]
<p>This looks good Ryan. My comments are minor. Thanks for quickly fixing that issue with the previous lab so this will work correctly when deployed. I was lost with error message. </p>

File: courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb (hunk 1 above)
Comment (ryangillard) [reply courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>Ya, once these are all done I want to do another pass on these intros. I was originally following a template, but it is redundant I think. Then again, do students even read this section haha?</p>

Comment (ryangillard) [reply courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>This is a python variable, above is a bash variable. Different scopes.</p>

Comment (ryangillard) [reply courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>I'll add a #TODO. By now, hopefully students will know the drill haha.</p>

Comment (ryangillard) [reply courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>PROJECT is not being used anywhere in bash below, did it not run without it for you? I tested specifically for this earlier and had no issues.</p>

Comment (ryangillard) [reply courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>Fixed</p>

File: courses/machine_learning/deepdive2/structured/solutions/5b_deploy_keras_ai_platform_babyweight.ipynb (hunk 2 above)
Comment (ryangillard) [reply courses/machine_learning/deepdive2/structured/solutions/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>Same as above.</p>

File: courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb (hunk 1 above)
Comment (ryangillard) [reply courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>Done</p>

File: courses/machine_learning/deepdive2/structured/solutions/5b_deploy_keras_ai_platform_babyweight.ipynb (hunk 2 above)
Comment (ryangillard) [reply courses/machine_learning/deepdive2/structured/solutions/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>Done</p>

Comment (ryangillard) [reply courses/machine_learning/deepdive2/structured/solutions/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>Same as above.</p>

Comment (ryangillard) [reply courses/machine_learning/deepdive2/structured/solutions/5b_deploy_keras_ai_platform_babyweight.ipynb]: <p>Fixed</p>

**Comment by ryangillard** [comment]
<p>Thanks Mike for the great review! I've responded to all of your comments and have made all changes, so the ball's back to you. :)</p>

File: courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb (hunk 1 above)
Comment (munnm) [reply courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: ok

Comment (munnm) [reply courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: right. thanks.

Comment (munnm) [reply courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb]: ah yeah. Just added it out of habit, but yeah not needed later.

**Pull request merged**